flutter_wasm/example/pubspec.yaml

```diff
@@ -11,15 +11,25 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_wasm:
-    path: ../
+    git:
```

Comment:
just leave this as `flutter_wasm: any` and below!

Reply:
Done, for this file only. Should I switch over the other pubspec too?

Reply:
Absolutely. Or just put in the version you think it'll be published as.

Either is better than `git:`!!